pkg/bootstrap/bootstrap.go

```diff
@@ -188,14 +188,30 @@ func (c *BootstrapClient) applyManifests(ctx context.Context, objs []unstructure
 		}
 
 		bar.Describe(fmt.Sprintf("Applying %v (%v)", obj.GetName(), obj.GetKind()))
-		if err = retry.RetryOnConflict(retry.DefaultRetry, func() error {
-			_, err = c.client.Resource(mapping.Resource).Namespace(obj.GetNamespace()).
-				Apply(ctx, obj.GetName(), &obj, metav1.ApplyOptions{Force: true, FieldManager: "glasskube"})
-			return err
-		}); err != nil {
-			return err
+		if obj.GetKind() == "Job" {
+			options := metav1.DeletePropagationBackground
+			fmt.Println("Deleting Job")
+			err := c.client.Resource(mapping.Resource).Namespace(obj.GetNamespace()).Delete(
+				ctx,
+				obj.GetName(),
+				metav1.DeleteOptions{PropagationPolicy: &options})
+			if err != nil {
+				return err
+			}
+			_, err = c.client.Resource(mapping.Resource).Namespace(obj.GetNamespace()).Create(ctx, &obj, metav1.CreateOptions{})
```

Comment:
I think we should use `Apply` instead of create here as well, since this is also done in the `else` branch.

Reply:
Oh, but `Apply` just updates the current resources instead of creating them?

Reply:
It can do both :slightly_smiling_face: